ShowSQLTables/PrinterOfList.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ShowSQLTables
+{
+    class PrinterOfList
+    {
+        private DataTable dataTable;
+
+        public PrinterOfList (DataTable dataTbl)
+        {
+            this.dataTable = dataTbl;
+        }
+
+        public void Print()
```

Comment:
If I say you need to use the BookPrice and show avg price of it  how this structure will handle it?